datafusion-examples/examples/dataframe.rs

```diff
@@ -41,12 +44,20 @@ async fn main() -> Result<()> {
     // print the results
     df.show().await?;
 
+    // create a csv file waiting to be written
+    let path = "example.csv";
+    let dir = tempdir()?;
+    let file_path = PathBuf::from(dir.path()).join(path);
```

Comment:
You might be able to express this more concisely like this, perhaps

```suggestion
    let file_path = dir.path().join("example.csv");
```